api/src/lib/sqs.ts

```diff
@@ -0,0 +1,50 @@
+import AWS from 'aws-sdk';
+
+const queueUrl =
+    process.env.STAGE === 'local'
+        ? 'http://localhost:4566/000000000000/science-octopus-pdf-queue-local'
+        : `https://sqs.eu-west-1.amazonaws.com/948306873545/science-octopus-pdf-queue-${process.env.STAGE}`;
+
+const config = {
+    region: 'eu-west-1',
+    endpoint: process.env.STAGE === 'local' ? process.env.LOCALSTACK_SERVER : 'https://sqs.eu-west-1.amazonaws.com'
```

Comment:
As I mentioned above, checking the STAGE wouldn't be necessary if we have an SQS_ENDPOINT env var